lib/improver/cube_combiner.py

```diff
@@ -44,6 +43,17 @@ class CubeCombiner(BasePlugin):
 
     """
 
+    COMBINE_OPERATORS = {
```

Comment:
Perhaps the docstring of the `CubeCombiner` class or possibly the `process` method could be extended to indicate what functionality `CubeCombiner` has, compared to the circumstance when adding two cubes using e.g. `np.add` might be more appropriate, so that it's clear what the use case is for `CubeCombiner` (e.g. checking that dimensions match and handling of adjustments to bounds) vs just using e.g. `np.add`.

Reply:
I'm struggling to see how this is unclear.  It's the concept of combining cubes (which are inherently different datasets: either different diagnostics, representing a different time or from a different forecast cycle), vs making a correction to an existing dataset (such as adjusting precipitation rates for orographic enhancements).  This isn't something I think belongs in a docstring, I'm afraid.